### PR TITLE
[Hyperdrive] Minor improvements to Get started page

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**

### CODE REVIEW

File: content/hyperdrive/get-started.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ pcx_content_type: get-started
 
 # Get started
 
-Hyperdrive dramatically speeds up access to your existing databases from Cloudflare Workers, making even single-region databases feel globally distributed.
+Hyperdrive accelerates access to your existing databases from Cloudflare Workers, making even single-region databases feel globally distributed.
 
-- By maintaining a connection pool to your database within Cloudflare's network, Hyperdrive cuts out what can typically be seven round-trips to your database before you can even send a query: the TCP handshake (1x), TLS negotiation (3x) and database authentication (3x).
-- On top of that, Hyperdrive understands the difference between read and write queries to your database, and can intelligently cache the most common read queries made: improving both query performance and reducing load on your origin database.
+By maintaining a connection pool to your database within Cloudflare's network, Hyperdrive reduces seven round-trips to your database before you can even send a query: the TCP handshake (1x), TLS negotiation (3x) and database authentication (3x).
```
Comment (RebeccaTamachiro) [inline content/hyperdrive/get-started.md]:
```suggestion
By maintaining a connection pool to your database within Cloudflare's network, Hyperdrive reduces seven round-trips to your database before you can even send a query: the TCP handshake (1x), TLS negotiation (3x), and database authentication (3x).
```

File: content/hyperdrive/get-started.md [hunk 2]
```diff
@@ -6,10 +6,11 @@ pcx_content_type: get-started
...
+
+Hyperdrive understands the difference between read and write queries to your database, and can cache the most common read queries made, improving query performance and reducing load on your origin database.
```
Comment (RebeccaTamachiro) [inline content/hyperdrive/get-started.md]:
```suggestion
Hyperdrive understands the difference between read and write queries to your database, and can cache the most common read queries, improving performance and reducing load on your origin database.
```

File: content/hyperdrive/get-started.md [hunk 3]
```diff
@@ -21,7 +22,7 @@ This guide will instruct you through:
 
 {{<Aside type="note" header="Workers Paid plan required">}}
```
Comment (RebeccaTamachiro) [inline content/hyperdrive/get-started.md:23]:
```suggestion
{{<Aside type="note" header="Workers paid plan required">}}
```

File: content/hyperdrive/get-started.md [hunk 4]
```diff
@@ -21,7 +22,7 @@ This guide will instruct you through:
...
 
-Hyperdrive is available to all users on the [Workers paid plan](/workers/platform/pricing/#workers).
+Hyperdrive is available to all users on the [Workers Paid plan](/workers/platform/pricing/#workers).
```
Comment (RebeccaTamachiro) [inline content/hyperdrive/get-started.md:25]:
```suggestion
Hyperdrive is available to all users on the [Workers paid plan](/workers/platform/pricing/#workers).
```

Comment (RebeccaTamachiro) [reply content/hyperdrive/get-started.md:25]: Suggesting to revert this change and, instead, make the Aside header follow what is in the content. Ignore it if, for some reason, `Workers Paid plan` was agreed to always be written that way, but searching for it within the Dev Docs project, it seems it's written in both ways and the lower case option would be more in line with the Style Guide.

**Review by RebeccaTamachiro — APPROVED** [review]

File: content/hyperdrive/get-started.md (hunk 4 above)
Comment (Maddy-Cloudflare) [reply content/hyperdrive/get-started.md:25]: Yes, Workers Paid plan was suggested to be written that way. Also came to know that the Workers pricing model is under review. I'll keep it capital for now. :)

**Pull request merged**